src/app/container/info-tab/info-tab.component.css

```diff
@@ -13,3 +13,7 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+
+.icon-display {
```

Comment:
more descriptive name, possibily move the styles.scss to prevent duplicate class